CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ##### Enhancements
 
+* Add `ExplicitSelfRule` Opt-In rule that enforces the use of `self.` for instance members
```

Comment:
80 char hard-wrap, trailing period following by two spaces. Format described here: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes

Also, we consistently refer to "opt-in" in lowercase throughout the changelog.